src/nanoFramework.Graphics/TouchPanel/Core/TouchPanel.cpp

```
@@ -165,7 +165,7 @@ HRESULT TouchPanelDriver::GetDeviceCaps(unsigned int iIndex, void *lpOutput)
             return FALSE;
     }
 
-    return TRUE;
+    return true;
```

Clang (14.0.6)

[warning] implicit conversion bool -> 'HRESULT' (aka 'int')

(readability-implicit-bool-conversion)